examples/dreambooth/train_dreambooth_lora_sdxl.py

```diff
@@ -453,6 +455,10 @@ def __init__(
         self.instance_unet_added_conditions = instance_unet_added_conditions
         self.class_unet_added_conditions = class_unet_added_conditions
 
+        if tokenizers is not None:
+            self.tokenizer_one = tokenizers[0]
+            self.tokenizer_two = tokenizers[1]
+
```

Comment:
if we know the tokenizers are going to be passed as a list of two elements, we should just pass both tokenizers as separate arguments